### ISSUE TITLE
how to run it on MacOSX?

### ISSUE THREAD

**Issue by huary** [tzutalin/labelImg#181]
when I run it like that:
$make qt4py2
pyrcc4 -py2 -o resources.py resources.qrc
$python labelImg.py
Segmentation fault: 11

- **OS:**
- **PyQt version:**

**Comment by suessmann** [comment]
same issue here!

**Comment by RodolfoFerro** [comment]
Hey @huary, @suessmann, this worked for me:

### Python 3 + Qt5 (Works on macOS High Sierra)

```bash
brew install qt  # will install qt-5.x.x
brew install libxml2
make qt5py3
python labelImg.py
python  labelImg.py [IMAGE_PATH] [PRE-DEFINED CLASS FILE]
```

**Comment by vdalv** [comment]
Please follow the [macOS instructions](https://github.com/tzutalin/labelImg#macos), and let us know if you have any issues.

Thanks,

-Vlad

**Comment by duchengyao** [comment]
@kuabhish 
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
``` 

before running `labelImg`

**Comment by tzutalin** [comment]
@kuabhish 
Fixed your problem on d27e6a5

**Comment by maharshi4697** [comment]
Hi, when I try to run this on my mac I get this error
File "/Users/maharshichattopadhyay/Downloads/labelImg-master/labelImg.py", line 29, in <module>
    import resources
ImportError: No module named resources
